elections/tac-and-scir-election-process.md

```diff
@@ -0,0 +1,66 @@
+The voter eligibility form asks you for an example of your contributions; this is merely to make it easier for the Election Officials and OpenSSF staff to validate eligibility.
+If you have in any way been involved in or care about OpenSSF, but are in doubt as to whether your contribution “counts”, please fill it out anyways, and we will follow up.
+
+[Voter Eligibility Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLSegUTOrxrGPVwDrPH9RARmYxX5YEkW3j67RU4uWABkKD5x8og/viewform)
+
+ 
+## TAC Self-Nomination Process
+
+The OpenSSF Technical Advisory Council (TAC) is composed of seven total individuals, four of whom are elected annually.
+Eligible community members, according to the same criteria defined above for voter eligibility, can self-nominate using the form below.
+
+[TAC Candidate Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLSdxUbbfJGgjU_cZ_uGvUm-5ePoXu7q2pGiNyH2G8euwpn23Sw/viewform)
+
+
+## SCIR Self-Nomination Information and Process
+
+Since early in its existence, the OpenSSF Governing Board has sought to ensure it gets adequate input from voices in the software security community who would otherwise not be at the table.
+We seek candidates for the Security Community Individual Representative (SCIR) who can represent those voices, while also being a subject matter expert in the field with their own set of perspectives.
+Familiarity with the different OpenSSF working groups and projects, and being able to dedicate the time to be sufficiently informed on the issues that arise in our monthly calls and ongoing discussions, is highly desired.
+It is also highly desired, but not required, that the SCIR be an eligible OpenSSF community member by the criteria defined above.
+
+[SCIR Candidate Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLScoabVVjW1kvTNcKXhTEdvxQd13Caql-I36MOixHVO1dLwIiQ/viewform)
+
+
+## Election Process
+
+The Election Process will be monitored by Election Officials, who
+* review nominees and confirm their eligibility,
+* approve the election timeline, and
+* review the ballot request form and validate voter eligibility.
+
+Election Officials are eligible community members who have volunteered to oversee and help ensure the integrity of the election process. By volunteering in this role, they recuse themselves from running for a TAC or SCIR position but they still are allowed to vote in the election.
+
+Candidates for each election will be reviewed by the Election Officials to ensure their eligibility per the guidelines above. At the end of the validation period, the complete list of candidates will be published in this repo.
+
+Similarly, the pool of eligible voters will be reviewed by Election Officials to ensure that all voters meet the eligibility criteria established above If a ballot requester is considered ineligible, an Election Official will follow up directly with the individual.
+
```

Comment:
```suggestion
If the number of nominees dose not meet the minimum number of nominees needed to fill the open positions, ...
```

Reply:
I'm not sure what would happen in this instance, but it should be defined. @jorydotcom or @brianbehlendorf do you know if this has already been established somewhere and I missed it?

Reply:
@bennyvasquez in that case, we would appoint the nominees and open a second nomination process to fill the remaining roles. If we had 4 eligible nominees and 4 open positions, then those individuals would be elected unopposed.

Reply:
@bennyvasquez, @jorydotcom: I included some wording for this in the latest commit https://github.com/ossf/tac/pull/139/commits/19c8f811bf8823582aa08d722bc540d36c16b631